src/AcmCountCard/AcmCountCard.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import '@patternfly/react-core/dist/styles/base.css'
+import React from 'react'
+import { Meta } from '@storybook/react'
+import { AcmCountCard, loadingCard } from './AcmCountCard'
```

Comment:
`AcmSearchCard` sounds more intuitive.  Just a suggestion.

Reply:
Was trying to be more generic as this card conditionally covers multiple use cases

Reply:
Yes we will also be using this component

Reply:
Ok, I'll have to get used to the new name!  😄